app/serializers/credit_card_serializer.rb

```diff
@@ -1,3 +1,3 @@
 class CreditCardSerializer < ActiveModel::Serializer
-  attributes :id, :month, :year, :number, :first_name, :last_name
+  attributes :id, :month, :year, :card_number, :first_name, :last_name, :cvv, :last_four
```

Comment:
technically this would always be empty right? since card number isn't saved in the database